requirements.in

```
@@ -18,4 +18,4 @@ marshmallow==3.22.0
 pydantic==2.10.3
 pyOpenSSL==24.2.1
 pytz==2024.2
-signxml==3.2.2
+signxml==4.0.0
```

See: https://github.com/cordada/lib-cl-sii-python/pull/736#discussion_r1880349409

```suggestion
signxml==4.0.3
```